MediaGalleryUi/view/adminhtml/web/js/image/image-actions.js

```diff
@@ -74,15 +74,18 @@ define([
         saveImageDetailsAction: function () {
             var saveDetailsUrl = this.mediaGalleryEditDetails().saveDetailsUrl,
                 modalElement = $(this.modalSelector),
-                dataForm = modalElement.find('#image-edit-details-form');
+                imageId = this.imageModel().getSelected().id;
 
-            if (dataForm.validation('isValid')) {
+            modalElement.find('#image-edit-details-form').validation();
```

Comment:
@yolouiese Are the validation changes relevant to this pull request?

Reply:
Sorry, it was not supposed to be included in the pull request. Maybe it got included after I did git pull origin to update my branch. I'll update again my pull request.